CMakeLists.txt

```diff
@@ -79,7 +79,6 @@ function(default_compile_options TARGET)
       target_compile_options(${TARGET} PRIVATE -fno-omit-frame-pointer)
     endif()
     if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-      target_compile_options(${TARGET} PRIVATE -fcolor-diagnostics)
```

Comment:
I mentioned this in the commit message. This is clang's default argument for terminals that support color. Terminals which do not support color display ASCII escape codes if that option is included. This was causing issues in Emacs which automatically converts the line output into links so color is not necessary.